contexts/did-v1.jsonld

```diff
@@ -13,6 +13,7 @@
     "Ed25519VerificationKey2018": "sec:Ed25519VerificationKey2018",
     "RsaVerificationKey2018": "sec:RsaVerificationKey2018",
     "SchnorrSecp256k1VerificationKey2019": "sec:SchnorrSecp256k1VerificationKey2019",
+    "X25519KeyAgreementKey2019": "sec:X25519KeyAgreementKey2019",
```

Comment:
I would've +1'd this before the big discussion on VerificationKey vs. PublicKey vs. JsonWebKey... I suggest we hold off on this until we have a clear direction on https://github.com/w3c/did-spec-registries/pull/96.

Reply:
I'd suggest merging it, since this is unstable, and this causes JSON-LD parsing errors.... terms like this are dropped in signatures... starting to feel like this might be a best practice : ).... I'm joking... its a security issue.... You will see `did:/X25519KeyAgreementKey2019` in Linked Data Signatures... its in the wild, its just not documented... this corrects that... better thing to do would be add a deprecation warning... when thats advisable.

cc @dlongley 

I would be in favor of adding language elsewhere (in html) that helps users who want to read documentation... this fixes a security issue (unknown type properties are dropped).